fuuk/people/tests/test_models.py

```diff
@@ -98,3 +114,20 @@ def test_file_unicode(self):
         self.assertEqual(attachment.file.read(), b'content')
         self.assertTrue(os.path.isfile(os.path.join(self.tmp_dir, 'files/courses/sc-rze.txt')))
         attachment.file.close()
+
+
+class TestCVFilename(SimpleTestCase):
+    """
+    Test `cv_filename` function.
+    """
+    def test_cv_filename(self):
+        # Test `cv_filename` function
+        self.assertEqual(cv_filename(sentinel.instance, 'foo.dat'), 'cv/foo.dat')
+
+
+class TestImageFilename(TestCase):
+    """
+    Test `image_filename` function.
+    """
+    def test_image_filename(self):
+        self.assertEqual(image_filename(sentinel.instance, 'foo.dat'), 'img/person/foo.dat')
```

Comment:
We should ensure that these functions sanitizes the inputs. It doesn't have to be thorough, but at least one check.